### PR TITLE
feat: update additional info

### PR DESCRIPTION
**Pull request by ReedSoftware**

### CODE REVIEW

File: server/routes/journeys/referral/additional-information/controller.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ export class ReferralAdditionalInformationController {
     res.render('referral/additional-information/view', {
       otherInformation,
       backUrl: 'safer-custody',
+      maxLengthChars: 4000,
+      threshold: '75',
```
Comment (bill-moj) [inline server/routes/journeys/referral/additional-information/controller.ts:12]: i just have a thought maybe it's better to have `maxLengthChars or 4000` and `threshold or "75"` in the view.njk?
but maybe not.

File: server/routes/journeys/update-referral/additional-information/controller.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { NextFunction, Request, Response } from 'express'
+import { SchemaType } from './schemas'
+import { MESSAGE_REFERRAL_ADDITIONAL_INFO_UPDATED, PatchReferralController } from '../../base/patchReferralController'
+import { generateSaveTimestamp, getMaxCharsAndThresholdForAppend } from '../../../../utils/appendFieldUtils'
+
+export class UpdateAdditionalInfoController extends PatchReferralController {
+  GET = async (req: Request, res: Response) => {
+    res.render('referral/additional-information/view', {
+      isProactiveReferral: req.journeyData.referral!.isProactiveReferral,
```
Comment (bill-moj) [inline server/routes/journeys/update-referral/additional-information/controller.ts:9]: i don't think this property is need for the view file

File: server/routes/journeys/update-referral/additional-information/schemas.ts [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+export const schemaFactory = async (req: Request, res: Response) => {
+  const maxLengthChars = req.journeyData.isUpdate
+    ? getMaxCharsAndThresholdForAppend(
+        res.locals.user.displayName,
+        req.journeyData.csipRecord?.referral.otherInformation,
+      ).maxLengthChars
+    : 4000
+
+  const ADDITIONAL_INFORMATION_MSG = 'Enter an update to the additional information'
+
```
Comment (bill-moj) [inline server/routes/journeys/update-referral/additional-information/schemas.ts:17]: this variable can be declared outside of the function call

File: server/routes/journeys/referral/additional-information/controller.ts (hunk 1 above)
Comment (ReedSoftware) [reply server/routes/journeys/referral/additional-information/controller.ts:12]: I thought about it, but it isn't really a view concern, it's setting the values, and I think it makes sense to be more explicit about it in the controller.

File: server/routes/journeys/update-referral/additional-information/controller.ts (hunk 2 above)
Comment (ReedSoftware) [reply server/routes/journeys/update-referral/additional-information/controller.ts]: True, removed

File: server/routes/journeys/update-referral/additional-information/schemas.ts (hunk 3 above)
Comment (ReedSoftware) [reply server/routes/journeys/update-referral/additional-information/schemas.ts]: Done

**Review by bill-moj — APPROVED** [review]

**Pull request merged**